src/jv.c

```diff
@@ -1862,6 +1862,68 @@ jv jv_object_iter_value(jv object, int iter) {
 /*
  * Memory management
  */
+jv jv_unshare(jv input){
```

Comment:
Not sure how ugly but i guess one could save on `jv_free` calls by having different ownership rule for unshare? too big foot gun?

Reply:
even within the implementation of `jv_unshare()` we benefit from it dereferencing its input. We could have `jv_unshare()` not consume its memory, but at the cost of having to write a lot of stuff to buffers

Reply:
Is `jv_unshare()` supposed to consume a reference to its argument?  I think it should.